internal/pkg/manifest/validate.go

```diff
@@ -160,6 +183,9 @@ func (l LoadBalancedWebServiceConfig) Validate() error {
 // Validate returns nil if BackendService is configured correctly.
 func (b BackendService) Validate() error {
 	var err error
+	if err = b.DeployConfig.Validate(); err != nil {
+		return fmt.Errorf(`validate "Deployment Configuration": %w`, err)
```

Comment:
```suggestion
		return fmt.Errorf(`validate "deployment": %w`, err)
```
and same for the other places!